packages/handlers/openapi/src/openapi-to-graphql/resolver_builder.ts

```diff
@@ -475,12 +476,7 @@ export function getResolver<TSource, TContext, TArgs>(
       };
     }
 
-    for (const paramName in query) {
-      const val = query[paramName];
-      if (val !== undefined) {
-        urlObject.searchParams.set(paramName, val);
-      }
-    }
+    urlObject.search = qs.stringify(query);
```

Comment:
This overwrites existing values

Reply:
maybe?

```suggestion
    Object.assign(urlObject.search, qs.stringify(query));
```

Reply:
search is not an object, it is a string.

Reply:
oof, right!